template/packages/backend/src/conductor.ts

```diff
@@ -0,0 +1,57 @@
+import {UserSignup, User} from '@/lib';
+import * as Auth from '@/auth';
+import * as D from 'io-ts/lib/Decoder';
+
+import {makeExpress} from '@/backend/src/utils';
+import {parseEnv} from '@/backend/src/env';
+import {App} from '@/backend/src/app';
+import {DB} from '@/backend/src/db';
+
+const signupDecoder: D.Decoder<unknown, UserSignup> = D.type({
+  name: D.string,
+});
+
+function makeEnv() {
+  const env = parseEnv();
+  if (typeof env == 'string') {
+    console.error('missing or invalid keys in ENV');
+    console.error('==============================');
+    console.error('');
+    console.error(env);
+    console.error('');
+    console.error('unsolicited advice: check your packages/backend/.env file');
+    process.exit(1);
+  }
+  return env;
+}
+
+export class Conductor {
+  env = makeEnv();
+  authController = new Auth.AuthController<UserSignup, User>(signupDecoder);
+  db = new DB(this.env);
+
```

Comment:
A makeshift dependency injector so we don't have to wholeheartedly commit to Loopback just quite yet